docs/usage/shoot_serviceaccounts.md

```diff
@@ -40,33 +40,6 @@ This can be ensured by using [projected token volumes](https://kubernetes.io/doc
 Additionally, all [`ServiceAccount` token secrets](https://kubernetes.io/docs/concepts/configuration/secret/#service-account-token-secrets) should be recreated.
 Apart from this you should wait for at least `12h` to make sure the control plane and system components receive a new token from Gardener.
 
-## Signing Key Secret
```

Comment:
https://github.com/gardener/gardener/blob/a7d40c36bc7dfe6c50dc87b6021c11b936f43a01/docs/usage/shoot_serviceaccounts.md#L16-L18 could be removed as well.